lib/carthage_cache/configuration.rb

```diff
@@ -32,7 +32,7 @@ def self.default
           secret_access_key: ENV['AWS_SECRET_ACCESS_KEY'],
           profile: ENV['AWS_PROFILE']
         },
-        tmpdir: Dir.tmpdir
+        tmpdir: File.join(Dir.home, 'Library', 'Caches')
```

Comment:
That's a good observation :) This behaviour is already implemented though ☺️

In fact, `Project` creates the cache directory [here](https://github.com/iflix-letsplay/carthage_cache/blob/local-cache/lib/carthage_cache/project.rb#L45) using the `cache_dir_name` parameter. The value of `cache_dir_name` is set [at initialization time](https://github.com/iflix-letsplay/carthage_cache/blob/local-cache/lib/carthage_cache/project.rb#L11). `Application` [initialises `Project`](https://github.com/iflix-letsplay/carthage_cache/blob/local-cache/lib/carthage_cache/application.rb#L15) passing `CACHE_DIR_NAME` as the name the value for the `cache_dir_name` parameter. [The value of `CACHE_DIR_NAME` is `"carthage_cache"`](https://github.com/iflix-letsplay/carthage_cache/blob/local-cache/lib/carthage_cache/application.rb#L7).

This all results into `~/Library/Caches/carthage_cache` being created, which is the behaviour we want 😄